src/dpq2/conv/net.d

```diff
@@ -0,0 +1,76 @@
+module dpq2.conv.net;
+
+import dpq2.conv.to_d_types;
+import dpq2.oids : OidType;
+import dpq2.value;
+
+import std.bitmanip : bigEndianToNative;
+import std.conv : to;
+import std.exception : enforce;
+import std.format : format;
+import std.socket : AddressFamily;
+import std.traits : hasMember;
+
+
+enum PG_TYPE : ushort {
+	INET_ADDR = 0,
+	CIDR_ADDR = 1
+}
+
+template isNetworkAddress(T) {
+	enum isNetworkAddress = hasMember!(T, "isInet");
+	//pragma(msg, "isNetworkAddress!" ~ T.stringof ~ " = " ~ isNetworkAddress.to!string);
```

Comment:
Can be removed?

Reply:
If you point to line 22, freely dismiss, I forgot to remove it myself before pushing. Nevertheless, the template isNetworkAddress itself is essential for binarayValueAs template later in the code.

Reply:
I have removed it in the follow-up commits.